src/components/Button/Button.less

```diff
@@ -1,97 +1,143 @@
 // button sizes
-@Button-size-padding: 12px;
+@Button-size-padding: 15px;
 @Button-size-lineHeight: 26px;
+@Button-size-height: 28px;
+@Button-size-height-small: 20px;
+@Button-size-height-large: 40px;
 @Button-size-shortlineHeight: 20px;
 @Button-size-borderRadius: @size-borderRadius;
 
 .lucid-Button {
 	.no-safari-flicker();
 	.box-sizing();
 	.gradient-animation(@featured-color-default-gradientStartColor, @featured-color-default-gradientEndColor);
+	height: @Button-size-height;
 	cursor: pointer;
 	display: inline-block;
 	outline: none;
 	border: 1px solid @featured-color-default-borderColor;
 	border-radius: @Button-size-borderRadius;
 	font: @font;
 	letter-spacing: .05em;
-	line-height: @Button-size-lineHeight;
 	padding: 0 @Button-size-padding;
 
+	& &-content {
+		display: inline;
+		> .lucid-Icon {
+
+		}
+		> .lucid-Icon {
+			margin-left: -3px;
+			margin-right: -3px;
+		}
+	}
+
 	// Types
-	&-primary {
+	&&-primary {
```

Comment:
does it make sense to create a higher-level mixin for the different _types_ since there's a lot of repeated style patterns here?